docs/fidesops/docs/guides/saas_postprocessors.md

```diff
@@ -0,0 +1,134 @@
+# SaaS Post-Processors
+
+Post-processors are, in essence, data transformers. Given data from an endpoint, we can add specific processors to transform the data into a format we need for subject requests.
+
+## Configuration
+
+Post-processors are configured within the `endpoints` section of a `saas_config`
+
+
+```yaml
+endpoints:
+  - name: messages
+    requests:
+      read:
+        path: /conversations/<id>/messages
+        request_params:
+          ...
+        postprocessors:
+          - strategy: unwrap
+            configuration:
+              data_path: conversation_messages
+          - strategy: filter
+            configuration:
+              field: from_email
+              value:
+                identity: email
+```
+
+Note: Order matters as its defined in the config. In the above example, unwrap will be run first, then the output of unwrap will be used in the filter strategy.
+
+
+## Supported Strategies
+- `unwrap`: Gets object at given data path.
+- `filter`: Filters data based on given field value.
```

Comment:
`unwrap` is great, but re: `filter` it's generally good not to use the verb we're describing in its own description